.github/workflows/zips.yml

```
@@ -232,12 +233,10 @@ jobs:
       - uses: actions/download-artifact@v3
         with:
           name: zip-files
-          path: ${{ github.ref }}
+          path: /tmp
       - name: Extract
         run: |
-          unzip google-site-kit.zip
-          mv google-site-kit/* .
-          rm -rf google-site-kit ./*.zip
+          unzip google-site-kit.zip -d ${{ github.ref }}
```

Hmm, it looks like we'd need to specify the full path to `google-site-kit.zip` here as follows:

```suggestion
          unzip /tmp/google-site-kit.zip -d ${{ github.ref }}
```

---

Thanks @techanvil, good catch, I've addressed the comment in an amended commit.